examples/connectivity/plot_probabilistic_atlas_extraction.py

```diff
@@ -64,4 +69,4 @@
 plotting.plot_connectome(correlation_matrix, coords,
                          edge_threshold="80%")
 
-plotting.show()
+plt.show()
```

Comment:
We use `plotting.show()` on purpose. In a lot of examples, this allows the user to display the plot without importing explicitely pyplot. We'd like to keep it this way.